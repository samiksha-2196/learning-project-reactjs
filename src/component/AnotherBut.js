//@flow
import React from 'react';
import PropTypes from 'prop-types';
import '../index.css';
const AnotherBut= props =>{
    return(
        <div className="activec" onClick={props.addTrip}></div>
        );
    };
    AnotherBut.propTypes={
        addTrip:PropTypes.any,
    };
    export default  AnotherBut;