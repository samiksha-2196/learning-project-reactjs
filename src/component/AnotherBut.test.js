//@flow
import React from 'react';
import renderer from 'react-test-renderer';
import AnotherBut from './AnotherBut.js';
test('AnotherBut when clicked',()=>{
    const component = renderer.create(<AnotherBut addTrip={true}/>);
    let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

}

)