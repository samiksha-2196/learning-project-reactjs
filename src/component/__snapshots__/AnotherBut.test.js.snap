// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnotherBut when clicked 1`] = `
<div
  className="activec"
  onClick={true}
/>
`;
